src/deroff.rs

```diff
@@ -1092,7 +1095,7 @@ impl Deroffer {
             }
             TblState::Data => {
                 if !self.tblTab.is_empty() {
-                    self.s = self.s.replace(&self.tblTab, "\n");
+                    self.s = self.s.replace(&self.tblTab, "\t");
```

Comment:
Oh, I did missed this. T_T

Reply:
I'm really glad i made that test based on the python lol